### DISCUSSION TITLE
page.to_image  Grey text disappeared

### DISCUSSION THREAD

**Discussion by yonglee7015** [jsvine/pdfplumber#918] [Ask for help with specific PDFs]
![image](https://github.com/jsvine/pdfplumber/assets/59268746/542ed2e9-1691-4192-af6f-71dae87029e8)

page.to_image()

Grey text disappeared, what should I do?

**Comment by jsvine** [comment]
Hi @yonglee7015, can you provide the PDF in question? Without the PDF, it will be difficult to help. But one thing you might want to try is to increase the resolution of the image conversion — i.e., `.to_image(resolution=150)`.
